src/snowflake/cli/_plugins/nativeapp/project_model.py

```diff
@@ -129,12 +134,15 @@ def project_identifier(self) -> str:
         # sometimes strip out double quotes, so we try to get them back here.
         return to_identifier(self.definition.name)
 
-    @cached_property
+    @property
     def package_name(self) -> str:
+        suffix = resource_suffix()
         if self.definition.package and self.definition.package.name:
-            return to_identifier(self.definition.package.name)
+            return append_to_identifier(
+                to_identifier(self.definition.package.name), suffix
```

Comment:
this is a bit tricky, because it assumes suffix is a safe identifier. 
concat_identifiers() might be what you're looking for.

Reply:
oh TIL, will update